bin/util

```diff
@@ -96,3 +96,15 @@ copy_directories() {
     fi
   done
 }
+
+curl_with_defaults() {
+  default_args=()
+
+  # Some company-internal users are building their slugs on CentOS where these newer curl commands aren't
+  # supported yet. This conditional ensures their builds continue to work.
+  if uname -a | grep -q Ubuntu; then
```

Comment:
I think you should instead check this way:

```
curl --help all | grep -q -- --retry-connrefused
```

Reply:
(first, it's more portable; second, Ubuntu Trusty which is still in ESM also has an old cURL without that flag)

Reply:
@Malax what David said, this is better than the version check I was writing.

Reply:
Yeah, agreed - this is much better and doesn't add complexity.